crates/fs/src/fs.rs

```diff
@@ -27,13 +27,15 @@ use futures::{future::BoxFuture, AsyncRead, Stream, StreamExt};
 use git::repository::{GitRepository, RealGitRepository};
 use gpui::{AppContext, Global, ReadGlobal};
 use rope::Rope;
+use serde::Deserialize;
+use serde_derive::Serialize;
```

Comment:
Is there a particular reason we need a separate `serde_derive` import instead of using `Serialize` from `serde` (which should already have the `derive` feature)?